client/components/PetTypeTile.js

```diff
@@ -1,10 +1,15 @@
 import React from "react"
+import { Link } from "react-router-dom"
 
 const PetTypeTile = props => {
   return (
     <div>
-      <h2>{props.type}</h2>
-      <img src={props.img}></img>
+      <Link to={`/pets/${props.type}`}>
+        <h2>{props.type}</h2>
+      </Link>
+      <Link to={`/pets/${props.type}`}>
```

Comment:
both of these Links look to be going to the same location, you can wrap both elements within the same Link!